src/app/public/modules/source-code/source-code.service.ts

```diff
@@ -26,7 +26,18 @@ export class SkyDocsSourceCodeService {
     return sourceCode
       .filter(file => file.filePath.indexOf(path) === 0)
       .map(file => {
-        file.rawContents = decodeURIComponent(file.rawContents);
+        // SKY UX 4 libraries will return rawContents encoded, while later libraries will
+        // return rawContents decoded. Check if the contents need decoded before returning them.
+        let decodedContents: string;
```

Comment:
Can you put a comment that communicates this branching logic can be removed for SKY UX 5? Something like...
```
TODO: Remove decoding after migrating to Angular CLI. Code examples will not be encoded for SKY UX 5 libraries.
```